File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGMeasureMode.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGMeasureMode {
    YGMeasureModeUndefined,
    YGMeasureModeExactly,
    YGMeasureModeAtMost;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGMeasureMode forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGMeasureMode.kt
package app.cash.redwood.yoga.internal.enums

enum class YGMeasureMode {
    YGMeasureModeUndefined, YGMeasureModeExactly, YGMeasureModeAtMost;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGMeasureMode {
            return values()[value]
        }
    }
}
